src/renderers/ascii_default/marks/mod.rs

```diff
@@ -0,0 +1,33 @@
+pub enum MarkKind {
+    Vertical,
+    Horizontal,
+    DownRight,
+    UpRight,
+    UpLeft,
+}
+
+#[cfg(not(feature = "unicode"))]
+impl MarkKind {
+    pub fn get(t: MarkKind) -> char {
+        match t {
+            MarkKind::Vertical => '|',
+            MarkKind::Horizontal => '-',
+            MarkKind::DownRight => '/',
+            MarkKind::UpRight => '|',
+            MarkKind::UpLeft => '^',
+        }
+    }
+}
+
+#[cfg(feature = "unicode")]
+impl MarkKind {
+    pub fn get(t: MarkKind) -> char {
+        match t {
+            MarkKind::Vertical => '│',
+            MarkKind::Horizontal => '─',
+            MarkKind::DownRight => '┌',
+            MarkKind::UpRight => '└',
+            MarkKind::UpLeft => '┘',
+        }
+    }
+}
```

Comment:
I don't think this kind of global selection is the correct use of cargo compilation features. This should be a choice at the point of rendering, not implicitly changed to use box drawing characters if anyone in the compilation graph turns on the feature.

Reply:
yep, I'm just testing if the API works at all to get such features. I won't land it as such.